pkg/downwardmetrics/downwardmetrics.go

```diff
@@ -31,3 +31,7 @@ func HasDownwardMetricDisk(vmi *v1.VirtualMachineInstance) bool {
 	}
 	return false
 }
+
+func HasDownwardMetricsChannel(vmi *v1.VirtualMachineInstance) bool {
```

Comment:
This func is very simple, better inline and drop it.

Reply:
Still valid.

Reply:
Apologizes, I've missed this comment last time. Sorry, but IMHO I think it's a bit easier to read in this way rather than in an inline function. WDYT?

Reply:
I would still prefer to inline it because it is just a one-liner.

Reply:
Done.